attributes/default.rb

```diff
@@ -93,7 +93,7 @@
 # KServe -> 0.10.0
 
 default['kube-hops']['kserve']['enabled']               = node['install']['kubernetes']
-default['kube-hops']['kserve']['version']               = "0.10.0"
+default['kube-hops']['kserve']['version']               = "v0.10.0"
```

Comment:
This won't work :) please remove the `v`. Otherwise there will be two v's in line 98 (img_tar_url)

Reply:
Or remove the `v` in line 98 instead

Reply:
I removed v in the image_tar_url